src/features/deposit/components/DepositForm/index.tsx

```diff
@@ -323,3 +334,11 @@ function getBlockchainsOptions(): Record<
 function shortenTxHash(txHash: string) {
   return `${txHash.slice(0, 5)}...${txHash.slice(-5)}`
 }
+
+function getDefaultBlockchainOption(token: SwappableToken) {
+  if (token && !isUnifiedToken(token)) {
+    const key = `${token.chainName}:${token.chainId}`
```

Comment:
This is important format, so I suggest leaving a comment about wtf this format means. Also suggest not using `chainId` at all.